flink-runtime/src/test/java/org/apache/flink/runtime/operators/CombineTaskTest.java

```diff
@@ -92,7 +92,35 @@ public void testCombineTask() {
 		
 		this.outList.clear();
 	}
-	
+
+	@Test
+	public void testOversizedRecordCombineTask() {
+		int keyCnt = 1;
+		int valCnt = 20;
+
+		addInput(new UniformRecordGenerator(keyCnt, valCnt, true));
+		addDriverComparator(this.comparator);
+		addDriverComparator(this.comparator);
+		setOutput(this.outList);
+
+		getTaskConfig().setDriverStrategy(DriverStrategy.SORTED_GROUP_COMBINE);
+		getTaskConfig().setRelativeMemoryDriver(combine_frac);
```

Comment:
I tried tweaking this to configure the memory size of the sorter, but failed. Please help too